src/stories/MessageBlock.stories.ts

```diff
@@ -0,0 +1,124 @@
+//TODO - storyshots currently don't seem to be working
+
+import Vue from 'vue'
+import Vuex from 'vuex'
+
+import MessageBlock from '@/components/interaction-designer/block-types/MobilePrimitives_MessageBlock.vue'
+import FlowBuilderSidebarEditorContainer from '@/stories/story-utils/FlowBuilderSidebarEditorContainer.vue'
+
+import {IRootState, store} from '@/store'
+import caseBlockStore, {BLOCK_TYPE as CASE_BLOCK_TYPE} from '@/store/flow/block-types/Core_CaseBlockStore'
+import messageBlockStore, {BLOCK_TYPE} from '@/store/flow/block-types/MobilePrimitives_MessageBlockStore'
+
+import { baseMounted, BaseMountedVueClass } from '@/stories/story-utils/storeSetup'
+
+import {Component} from 'vue-property-decorator'
+import {namespace} from 'vuex-class'
+import {IFlow, SupportedContentType, SupportedMode} from '@floip/flow-runner'
+import {get} from 'lodash'
+import {IResourceDefinitionVariantOverModesFilter} from '@/store/flow/resource'
+
+Vue.use(Vuex)
+
+const flowVuexNamespace = namespace('flow')
+
+export default {
+  title: 'MobilePrimitives/Message Block',
+  // Our exports that end in "Data" are not stories.
+  excludeStories: /.*Data$/,
+}
+
+const MessageBlockTemplate = `
+  <flow-builder-sidebar-editor-container :block="activeBlock">
+    <message-block 
+      :block="activeBlock" 
+      :flow="activeFlow"/>
+  </flow-builder-sidebar-editor-container>
+`
+
+@Component<any>(
+  {
+    components: {MessageBlock, FlowBuilderSidebarEditorContainer},
+    template: MessageBlockTemplate,
+    store: new Vuex.Store<IRootState>(store),
+    async mounted() {
+      await baseMounted.bind(this)(BLOCK_TYPE, messageBlockStore)
+    },
+  }
+)
+class DefaultClass extends BaseMountedVueClass {}
+// default log block state
+export const Default = () => (DefaultClass)
+
+@Component<any>(
+  {
+    components: {MessageBlock, FlowBuilderSidebarEditorContainer},
+    template: MessageBlockTemplate,
+    store: new Vuex.Store<IRootState>(store),
+    async mounted() {
+      const {block: {uuid: blockId}, flow: {uuid: flowId}} = await baseMounted.bind(this)(BLOCK_TYPE, messageBlockStore)
+
+      //TODO - support sending props to baseMounted?
+      this.block_setName({blockId: blockId, value: "A Name"})
+      this.block_setLabel({blockId: blockId, value: "A Label"})
+      this.block_setSemanticLabel({blockId: blockId, value: "A Semantic Label"})
+
+      // Set values on resource editor // TODO: find better way to do this once the resource editor is fully implemented
+      const {
+        languages: {
+          0: {id: languageId}
+        },
+      }: IFlow = this.activeFlow
+      const resourceId = get(this.activeBlock, `config.prompt`, '')
+
+      const variantSms: IResourceDefinitionVariantOverModesFilter = {
+        languageId,
+        modes: [SupportedMode.SMS],
+        contentType: SupportedContentType.TEXT,
```

Comment:
Would it be enough to just wrap contentType in an array here?

Reply:
resolved as discussed
- used an array `[contentType: SupportedContentType.TEXT]` instead
- then added `ts-ignore` & `TODO comment`